lib/classes/CLI.js

```diff
@@ -65,8 +65,14 @@ class CLI {
 
   generateMainHelp() {
     this.consoleLog('');
+    this.consoleLog(chalk.yellow('The Serverless Framework - https://serverless.com'));
+    this.consoleLog('');
+
+    this.consoleLog(chalk.yellow('Version:'));
```

Comment:
Version should imho be at the bottom of the help page because its not the most important content (But I really like having it there)